packages/wonder-blocks-core/util/enumerate-scroll-ancestors.js

```diff
@@ -0,0 +1,92 @@
+// @flow
+// Utility for finding the scroll ancestors of a child
+// Inspired by https://github.com/olahol/scrollparent.js
+// with modifications to make a generator that returns a sequence of all scroll
+// ancestors.
+// Also modified for our standards (and commented a bit).
+const style = function(node: Element, prop) {
+    return getComputedStyle(node).getPropertyValue(prop);
```

Comment:
Maybe `getStyleProp`, `style` as a verb indicates that's styling something which isn't the case.

Reply:
Fair.